docs/README.iOS.md

```diff
@@ -143,51 +143,66 @@ make -j$(getconf _NPROCESSORS_ONLN) -C tools/depends/target/binary-addons ADDONS
 
 ### 6.1. Build with Xcode
-Create an out-of-source build directory:
-```
-mkdir $HOME/kodi-build
-```
 
-Change to build directory:
+Change to Kodi's source code directory:
 ```
-cd $HOME/kodi-build
+cd $HOME/kodi
 ```
 
 Generate Xcode project for ARM 64bit (**recommended**):
 ```
-/Users/Shared/xbmc-depends/x86_64-darwin17.5.0-native/bin/cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos11.3_arm64-target-debug/share/Toolchain.cmake ../kodi
+make -C tools/depends/target/cmakebuildsys GEN=Xcode
 ```
 
-Or generate Xcode project for ARM 32bit:
-```
-/Users/Shared/xbmc-depends/x86_64-darwin15.6.0-native/bin/cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos9.3_armv7-target/share/Toolchain.cmake ../kodi
-```
+Additional cmake arguments can be supplied via the CMAKE_EXTRA_ARGUMENTS command line variable
+
+Example to generate Xcode project for ARM 32bit:
+````
+make -C tools/depends/target/cmakebuildsys GEN=Xcode CMAKE_EXTRA_ARGUMENTS="-DCMAKE_TOOLCHAIN_FILE=/Users/Shared/xbmc-depends/iphoneos9.3_armv7-target/share/Toolchain.cmake"
+````
 
-**WARNING:** The toolchain file location differs depending on your iOS and SDK version. You have to replace `x86_64-darwin15.6.0-native` and `iphoneos11.3_arm64-target-debug` or `iphoneos9.3_armv7-target` in the paths above with the correct ones on your system.
+**WARNING:** The toolchain file location differs depending on your iOS and SDK version. You have to replace `iphoneos11.3_arm64-target-debug` or `iphoneos9.3_armv7-target` in the paths above with the correct ones on your system.
 
 You can check `Users/Shared/xbmc-depends` directory content with:
 ```
 ls -l /Users/Shared/xbmc-depends
 ```
 
-**Start Xcode, open the Kodi project file** (`kodi.xcodeproj`) located in `$HOME/kodi-build` and hit `Build`.
+**Start Xcode, open the Kodi project file** (`kodi.xcodeproj`) located in `$HOME/kodi/build` and hit `Build`.
 
 **WARNING:** If you have selected a specific iOS SDK Version in step 4 then you might need to adapt the active target to use the same iOS SDK version, otherwise build will fail. Be sure to select a device configuration. Building for simulator is not supported.
 
 ### 6.2. Build with xcodebuild
-Alternatively, you can also build via Xcode from the command-line with `xcodebuild`, triggered by CMake:
+Alternatively, you can also build via Xcode from the command-line with `xcodebuild`:
+
+Change to Kodi's source code directory:
+```
+cd $HOME/kodi
+```
+
+Generate Xcode project for ARM:
+```
+make -C tools/depends/target/cmakebuildsys GEN=Xcode
+```
+
+Additional cmake arguments can be supplied via the CMAKE_EXTRA_ARGUMENTS command line variable
+
+An example to set -DCMAKE_TOOLCHAIN_FILE for an alternate sdk/ios version:
```

Comment:
bad example

Reply:
happy to remove it, but the example provided the 32bit option from existing docs.

Reply:
before it was needed, but cmakebuildsys auto sets it

Reply:
Ok, will remove 👍